src/services/SocketService/socketErrorService.js

```diff
@@ -0,0 +1,66 @@
+import camelcaseKeys from 'camelcase-keys';
+import store from 'app/store';
+import { errChat } from 'reducers/ChatDux';
+import Socket from './socketSetup';
+
+export const errChatRooms = () => {
```

Comment:
Please give better names... like `listenForErrors` or something. Same with the rest of the functions in this file.